CHANGELOG.md

```diff
@@ -1,5 +1,32 @@
 # Changelog
 
+## [0.12.2] - TBD
+
+### Added
+- AWS KMS server Keymanager plugin (#2066)
+- Upstream Authority plugin that uses the Certificate Authority Service from Google Cloud Platform (#2172)
+- Improved error returned during attestation of agents (#2159)
+- AWS IID node attestor now supports running in a location with no public internet access available for the server (#2119)
+- The k8s notifier can now rotate Webhook CA Bundles (#2022)
+- Rate limiting on X509 signing and JWT signing can now be disabled (#2142)
+- Uptime metrics in server and agent (#2032)
+- Timeouts to calls to KeyManager plugins (#2044)
+- Logging when lookup of user by uid or group by gid fails in the `unix` WorkloadAttestor plugin (#2048)
+
+### Changed
+- The k8s Workload Attestor now emits selectors for both image and image ID (#2116)
+- HTTP readiness endpoint on agent now checks the health of the Workload API (#2015, #2087)
+- SDS API in agent now returns an error if an SDS client requests resource names that don't exist (#2020)
+- Bundle and k8s-workload-registrar endpoints now only accept clients using TLS v1.2+ (#2025)
+
+### Fixed
+- SPIFFE ID equality check in CRD mode of the k8s-workload-registrar, that could lead to the issuance of new SVIDs even if that is not needed (#2155)
```

Comment:
Trying to think of how we can state this a little more succinctly...how about something like:

- Registration entry update handling in CRD mode of the k8s-workload-registrar to prevent unnecessary issuance of new SVIDs (#2155)